### ISSUE TITLE
`default: null` ignored

### ISSUE THREAD

**Issue by adrianwiechec-at-optilyz** [francescov1/mongoose-tsgen#142]
Another `default` problem, sorry :(

For:
```
import { Schema, model } from "mongoose";
const UserSchema = new Schema({
        nullableField: { type: String, required: true, default: null },
});
export const User = model("User", UserSchema);
```

`mtgen` gives us:
```
export type User = {
  nullableField: string;
  _id: mongoose.Types.ObjectId;
};
```
completely ignoring the possibility of `null`...

**Comment by francescov1** [comment]
Hey @adrianwiechec-at-optilyz, thanks for the submission!

So this behavior is technically correct with simple typescript configs, as a "null" value is allowed to be set for a type like string. But if the user has strictNullChecks enabled in their tsconfig, then you are right that this would be a bug.

Either way, I agree with you that this type output is not clear. It would be better if the resulting type was "string | null". Let me try to add it in this weekend.

**Comment by francescov1** [comment]
Fixed in v9.3.1 🚀

**Comment by adrianwiechec-at-optilyz** [comment]
Thank you, once again! :bow:
